### ISSUE TITLE
Controls working only on English layout

### ISSUE THREAD

**Issue by MrGenga** [MCMrARM/mcpelauncher-linux#98]
The title saying the issue

**Comment by MCMrARM** [comment]
I think I can confirm. Is this an issue in W10Ed?

**Comment by vzlomka76** [comment]
@MCMrARM no, i installed minecraft windows 10 edition and i don't have this problem. But i have this problem in Minecraft java
